Fix views bundler bugs and update dependencies (#1486)
rooseveltframework/roosevelt: CHANGELOG.md

## Next version

> - Fixed a bug where views bundler feature could fail to load in some rare instances.

Reviewer: ```suggestion
- Fixed a bug where the views bundler feature could fail to load in some rare instances.
```

## Next version

- Fixed a bug where views bundler feature could fail to load in some rare instances.
- Updated various dependencies.

## 0.24.0

> - Breaking: `startServer` and `initServer` methods are now async and need to be awaited to avoid possible race conditions.

Reviewer: ```suggestion
- Breaking: Altered `startServer` and `initServer` methods to now be async. They now need to be awaited to avoid possible race conditions.
```

>   - As a result starting the app in a single line needs to be updated to `await (require('roosevelt').startServer())`

Reviewer: ```suggestion
  - As a result, starting the app in a single line needs to be updated to `await (require('roosevelt').startServer())`.
```

  - Callback arguments have been removed from these methods as well.
- Breaking: Removed `app.get('routes')`, `onReqAfterRoute`, `onReqBeforeRoute`, `onReqStart`, and `onStaticAssetsGenerated`.
- Breaking: Renamed `app.httpServer` to `app.get('httpServer')` and `app.httpsServer` to `app.get('httpsServer')`.

> - Breaking `enable` param to `clientViews`.

Reviewer: ```suggestion
- Breaking: Added `enable` param to `clientViews`. If your app uses the `clientViews` feature, you will need to add this param to continue using the feature.
```

>   - In apps being migrated to this version this param will need to be added to turn the feature on.

Reviewer: ```suggestion
```
